### ISSUE TITLE
[Help]:  MAI or POT?

### ISSUE THREAD

**Issue by cyloveyou** [mobigroup/gmtsar#198]
Hello everyone，I wonder if pygmtsar could do MAI or POT?

**Comment by AlexeyPechnikov** [comment]
No, but you could create them yourself using PyGMTSAR-aligned data stacks.

**Comment by cyloveyou** [comment]
> No, but you could create them yourself using PyGMTSAR-aligned data stacks.

Thank you for your reply, AlexyPechnikov. As a college student and a beginner in InSAR, I would greatly appreciate any advice or assistance you could provide on learning MAI or POT. My email is xytxtoyou@gmail.com
